pxemgr/pxemanager.go

```diff
@@ -301,7 +335,23 @@ func (mgr *pxeManagerT) Start() error {
 		return err
 	}
 
-	return mgr.startIPXEserver()
+	go func() {
+		err := mgr.startIPXEserver()
+		if err != nil {
+			panic(err)
+		}
+	}()
+
+	go func() {
+		err := mgr.startAPIserver()
+		if err != nil {
+			panic(err)
+		}
+	}()
+
+	select {}
```

Comment:
I had expected you adjust either the interface or the go routine initialization before merging. Now you did nothing as it looks like. :D